feature-detects/emoji.js

```diff
@@ -10,12 +10,14 @@ Detects support for emoji character sets.
 define(['Modernizr', 'createElement', 'test/canvastext'], function( Modernizr, createElement ) {
   Modernizr.addTest('emoji', function() {
     if (!Modernizr.canvastext) return false;
+    var pixelRatio = window.devicePixelRatio || 1;
+    var offset = 8 * pixelRatio;
     var node = createElement('canvas'),
     ctx = node.getContext('2d');
```

Comment:
@ryanseddon I can help out with adding jscs - in master (and 0.12 which will be released soon) there's a new autofixing feature for whitespace which will help a lot with getting the codebase up to speed with the preferred style.